examples/cachestat.bpf.c

```diff
@@ -1,47 +1,66 @@
+// For kernel versions >= 5.16, use the instrumentation of tracepoint writeback_dirty_folio.
+// To do this:
+//   1. uncomment the sections:
+//     * OP_WDF in pache_cache_op struct
+//     * SEC("raw_tp/writeback_dirty_folio")
+//   2. comment out the sections:
+//     * OP_APD in pache_cache_op struct
+//     * SEC("fentry/account_page_dirtied")
+
 #include <vmlinux.h>
 #include <bpf/bpf_tracing.h>
 #include <bpf/bpf_core_read.h>
 #include "maps.bpf.h"
-#include "regs-ip.bpf.h"
+
+enum pache_cache_op {
+    OP_MPA,
```

Comment:
Let's make these a bit more descriptive and match with the actual label names.

Something like `OP_MPA` -> `OP_PAGE_ACCESS`.

Reply:
Corrected this.